src/lib/sdk.ts

```diff
@@ -57,7 +57,7 @@ class FlutterSDK {
   }
 
   private async _hasValidFlutterSdk(): Promise<boolean> {
-    return (await exists(this._sdkHome)) && (await exists(join(this._sdkHome, 'bin', 'flutter')));
+    return exists(this._sdkHome) && exists(join(this._sdkHome, 'bin', 'flutter'));
   }
```

Comment:
Why remove `await`

Reply:
`fs.exists` is deprecated so I'm now using `fs.existsSync` which is not